cli.js

```diff
@@ -3,6 +3,7 @@
 var fs = require('fs')
 
 var browserslist = require('./')
+var updateDB = require('./updater')
```

Comment:
The function should have the same name as a file.

I suggeest `updateDB = require('./update-db')` since we import function and it should be verb.